src/StatsGatherer.js

```diff
@@ -90,13 +90,25 @@ class StatsGatherer extends EventEmitter {
       const bitrate = Math.floor(8 * (bytes - previousBytesSent) / deltaTime);
 
       let lost = 0;
+      let sent = 0;
       if (report.remoteId && results[report.remoteId]) {
         lost = results[report.remoteId].packetsLost;
-      } else if (report.packetsLost) {
-        lost = parseInt(report.packetsLost, 10);
+      } else if (report.packetsLost || report.packetsSent) {
+        if (report.packetsLost) {
+          lost = parseInt(report.packetsLost, 10);
+        }
+        if (report.packetsSent) {
+          sent = parseInt(report.packetsSent, 10);
```

Comment:
Not sure how much guarding we want; but this can return NaN.    Thoughts on `parseInt(report.packetsSent, 10) || 0` ?